flaml/data.py

```diff
@@ -236,7 +237,7 @@ def fit_transform(self, X, y, task):
                     SimpleImputer(missing_values=np.nan, strategy='median'),
                     X_num.columns)])
                 X[num_columns] = self.transformer.fit_transform(X_num)
-            self._cat_columns, self._num_columns = cat_columns, num_columns
+            self._cat_columns, self._num_columns, self._datetime_columns = cat_columns, num_columns, datetime_columns
```

Comment:
Thank you for the contribution. A minor issue: this line is too long. Need to make it shorter
```suggestion
            self._cat_columns = cat_columns
            self._num_columns = num_columns
            self._datetime_columns = datetime_columns
```